ledger/caching/src/main/scala/com/daml/caching/DropwizardStatsCounter.scala

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.caching
+
+import com.daml.metrics.CacheMetrics
+import com.github.benmanes.caffeine.cache.stats.{CacheStats, StatsCounter}
+
+private[caching] final class DropwizardStatsCounter(
+    metrics: CacheMetrics,
+) extends StatsCounter {
+
+  override def recordHits(i: Int): Unit =
+    metrics.hitCount.inc(i.toLong)
+
+  override def recordMisses(i: Int): Unit =
+    metrics.missCount.inc(i.toLong)
+
+  override def recordLoadSuccess(l: Long): Unit =
+    metrics.loadSuccessCount.inc(l)
+
+  override def recordLoadFailure(l: Long): Unit =
+    metrics.loadFailureCount.inc(l)
+
+  override def recordEviction(): Unit = {
```

Comment:
This is deprecated. Override `recordEviction(int weight)` instead and increment the eviction count by one and eviction weight by the passed `weight`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6043/commits/516f1af59a19d50a52062d0ccffb1faa21e7f143

Reply:
I assume only one of these `recordEviction` methods will be called otherwise we would get evictions double-counted.